app/routers/users_router.py

```diff
@@ -32,11 +32,7 @@ async def get_all_users(skip: int = 0, limit: int = 100, db: Session = Depends(g
     return users
 
 
-@router.get("/users/me", response_model=UserResponse, responses={
-    401: {
-        "description": "Not authenticated, you must have Admin permission"
-    }
-})
```

Comment:
I don't think that's quite good either: you can't make a request, unless you're logged in.

In that case, you still need error handling for the case that someone requests that, despite being not logged in.

Reply:
the fast API already handles this error itself